src/main/java/org/broadinstitute/hellbender/engine/dataflow/coders/UUIDCoder.java

```diff
@@ -0,0 +1,28 @@
+package org.broadinstitute.hellbender.engine.dataflow.coders;
+
+import com.google.cloud.dataflow.sdk.coders.DelegateCoder;
+import com.google.cloud.dataflow.sdk.coders.StringUtf8Coder;
+
+import java.util.UUID;
+
+/**
+ * This is basically StringDelegateCoder, except that we need to call fromString instead of a one arg constructor.
+ */
+public class UUIDCoder {
```

Comment:
I think the advantages of having the coders clearly visible in their own classes/package outweighs any other considerations -- would rather not bury this within another class.